common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
You can make it a bit lower (like 30 pixels beyond the mask) to make it centered.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122810201-1e6e8980-d2e0-11eb-883e-8b94e62f3604.png)
![image](https://user-images.githubusercontent.com/46384992/122810210-23cbd400-d2e0-11eb-962b-4047efd46822.png)

</details>